packages/syntax-core/src/constants.ts

```diff
@@ -14,4 +14,14 @@ export const Color = [
   "white",
   "inherit",
 ] as const;
+
+export const BadgeColor = [
```

Comment:
@jliotta these `constants` should be for things that are truly global, not just for a specific component like `Badge`.